apps/server/src/modules/files-storage/files-storage-api.module.ts

```diff
@@ -18,6 +20,8 @@ import { FilesStorageUC } from './uc';
 		CoreModule,
 		HttpModule,
 		ConfigModule.forRoot(createConfigModuleOptions(config)),
+		InstanceConfigModule,
+		SchoolModule,
```

Comment:
The fileStorage mode should function independently of other services. It should resolve via “AuthorizationReferenceService”.